support/mender-inventory-provides

```diff
@@ -10,6 +10,31 @@ set -e
 # Also submit device_type, which is in a separate file, not in the database. We don't know if the
 # file contains a newline, so it's important that this is done last, to avoid corrupting other
 # entries.
-grep '^device_type=' /var/lib/mender/device_type
+
+# Set default values for configuration directories
+MENDER_CONF_DIR=${MENDER_CONF_DIR:-/etc/mender}
+MENDER_DATASTORE_DIR=${MENDER_DATASTORE_DIR:-/var/lib/mender}
+
+# Set default value for DeviceTypeFile
+default_device_type_file="/var/lib/mender/device_type"
```

Comment:
Please use UPPER_CASE or lower_case consistently for local variables

Reply:
@lluiscampos can you take a look at it now?